src/ApparentHorizons/ApparentHorizonComputeItems.hpp

```diff
@@ -0,0 +1,70 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <string>
+
+/// \cond
+class DataVector;
+/// \endcond
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/Tensor/EagerMath/DeterminantAndInverse.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/Tags.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Christoffel.hpp"
+#include "PointwiseFunctions/GeneralRelativity/ComputeGhQuantities.hpp"
+#include "PointwiseFunctions/GeneralRelativity/ComputeSpacetimeQuantities.hpp"
+#include "PointwiseFunctions/GeneralRelativity/IndexManipulation.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace ApparentHorizonComputeItems {
+template <size_t Dim, typename Frame>
+struct InverseSpatialMetric : gr::Tags::InverseSpatialMetric<Dim, Frame>,
```

Comment:
Our naming convention for `db::ComputeTag`s is  `BlaCompute` so in this case `InverseSpatialMetricCompute` so they show up in the documentation next to the corresponding `db::SimpleTag`s